easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iimpi-2020a-Python-3.8.2.eb

```
@@ -29,7 +29,7 @@ dependencies = [
     ('Qt5', '5.14.1'),
 ]
 
-configopts = "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts = '-DOPENGL_INCLUDE_DIR="$EBROOTMESA/include;$EBROOTLIBGLU/include" '
```

consistency across `CGAL` easyconfigs
```suggestion
configopts = r"-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
```